core/embed/rust/src/ui/util.rs

```diff
@@ -179,6 +180,32 @@ macro_rules! include_icon {
     };
 }
 
+pub const SLIDE_DURATION: Duration = Duration::from_millis(333);
```

Comment:
Let's be explicit with the unit `SLIDE_DURATION_MS`

Reply:
well the unit is in the constructor but :shrug: 87a76234de2eb419c85ce1ba025b208e6e9569ad